contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol

```diff
@@ -113,4 +113,14 @@ interface VRFCoordinatorV2Interface {
    * @param to - Where to send the remaining LINK to
    */
   function cancelSubscription(uint64 subId, address to) external;
+
+  /*
+   * @notice Check to see if there exists a request commitment consumers
+   * for all consumers and keyhashes for a given sub.
+   * @param subId where to send the funds
+   * @return exits true if outstanding requests
```

Comment:
Better IMO: `@return true if and only if there exist unfulfilled requests for the given subscription.`

Reply:
Updated